### PR TITLE
Add XML to LBFGS Maximum Entropy Classifier

### PR DESCRIPTION
**Pull request by wschin**
Toward #2522 following #3218.

### CODE REVIEW

File: src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs [hunk 1]
```diff
@@ -39,8 +39,63 @@
 
 namespace Microsoft.ML.Trainers
 {
-    /// <include file = 'doc.xml' path='doc/members/member[@name="LBFGS"]/*' />
-    /// <include file = 'doc.xml' path='docs/members/example[@name="LogisticRegressionClassifier"]/*' />
+    /// <summary>
+    /// The <see cref="IEstimator{TTransformer}"/> to predict a target using a linear logistic regression model trained with L-BFGS method.
```
Comment (wschin) [inline src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs]:
```suggestion
    /// The <see cref="IEstimator{TTransformer}"/> to predict a target using a maximum entropy multiclass classifier trained with L-BFGS method.
```

File: docs/api-reference/io-columns-multiclass-classification.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+### Input and Output Columns
+The input label column data must be key typed. This trainer outputs the following columns:
+
+| Output Column Name | Column Type | Description|
+| -- | -- | -- |
+| `Score` | array of <xref:System.Single> | The scores of all classes. Higher value means higher probability to fall into the associated class. If the i-th element has the lagest value, the predicted label index would be i. Note that i is zero-based index. |
```
Comment (shmoradims) [inline docs/api-reference/io-columns-multiclass-classification.md]: >array [](start = 12, length = 5)

if it's a vbuffer, let's call it 'vector' instead.

Comment (shmoradims) [inline docs/api-reference/io-columns-multiclass-classification.md]: >lagest [](start = 172, length = 6)

typo: largest

please use spell checker. we won't catch all the typos manually.
[https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellChecker](https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellChecker)

File: src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs [hunk 3]
```diff
@@ -39,8 +39,63 @@
...
+    /// L2-norm regularization is preferable for data that is not sparse and it largely penalizes the existence of large weights.
+    ///
+    /// An agressive regularization (that is, assigning large coefficients to L1-norm or L2-norm regularization terms) can harm predictive capacity by excluding important variables out of the model.
+    /// Therefore, choosing the right regularization coefficients is important when applying maximum entropy classifier.
+    /// ]]>
+    /// </format>
+    /// </remarks>
+    /// <seealso cref="Microsoft.ML.StandardTrainersCatalog.LbfgsMaximumEntropy(Microsoft.ML.MulticlassClassificationCatalog.MulticlassClassificationTrainers,System.String,System.String,System.String,System.Single,System.Single,System.Single,System.Int32,System.Boolean)"/>
+    /// <seealso cref="Microsoft.ML.StandardTrainersCatalog.LbfgsMaximumEntropy(Microsoft.ML.MulticlassClassificationCatalog.MulticlassClassificationTrainers,Microsoft.ML.Trainers.LbfgsMaximumEntropyMulticlassTrainer.Options)"/>
+    /// <seealso cref="Options"/>
```
Comment (shmoradims) [inline src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs]: please fix to cref style links provided by intellisens

File: src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs [hunk 4]
```diff
@@ -521,8 +521,8 @@ public IClassificationLoss CreateComponent(IHostEnvironment env)
         ///  Predict a target using a linear binary classification model trained with the <see cref="Trainers.LbfgsLogisticRegressionBinaryTrainer"/> trainer.
```
Comment (singlis) [inline src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs]: >   [](start = 11, length = 2)

Create <see cref="LbfgsLogisticRegressionBinaryTrainer"/>, which predicts a ...

Comment (wschin) [reply src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs]: This is binary classifier. I shouldn't touch it. Thanks!

---
In reply to: [277117848](https://github.com/dotnet/machinelearning/pull/3389#discussion_r277117848) [](ancestors = 277117848)

**Pull request merged**